isic/ingest/migrations/0009_auto_20220414_1543.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 4.0.3 on 2022-04-14 15:09
+
+from django.db import migrations
+
+
+def migrate_checks(apps, schema_editor):
+    Accession = apps.get_model('ingest', 'Accession')  # noqa: N806
+    User = apps.get_model('auth', 'User')  # noqa: N806
+    checks = ['quality_check', 'diagnosis_check', 'phi_check', 'duplicate_check', 'lesion_check']
+
+    for i, accession in enumerate(Accession.objects.iterator()):
+        if i % 10_000 == 0:
+            print(i)
+        check_values = [getattr(accession, check) for check in checks]
+
+        if False in check_values:
+            accession.qc_check = False
```

Comment:
How about:
* `accession.review_accepted: bool | None`
* `accession.review_time: datetime | None`
* `accession.review_user: User | None`

Reply:
Or a separate `Review` table with a `OneToOneField(Accession)`.

Reply:
Actually:
* Don't have a `accession.qc_check`
* Deduplicate entries in CheckLog (as you do here)
* Make `Checklogs.accession` a `OneToOneField` (will enforce the deduplication)
* Remove `Checklogs.change_field`
* Rename `Checklogs.change_to` -> `Checklogs.value` and make it non-nullable
* Rename `Checklogs` -> `Reviews`